Cargo.toml

```diff
@@ -35,3 +35,5 @@ regex = "1"
 glob = "0"
 notify = "4"
 shellexpand = "1"
+tokio = { version = "0.2.13", features = [ "full" ] }
```

Comment:
I'm afraid this might pull in lots lots of crates.

Reply:
Fixed!